tests/test_utils.py

```diff
@@ -10,6 +10,7 @@
 from betfair import models
 from betfair import constants
 from betfair.utils import BetfairEncoder
+from betfair.utils import make_payload
```

Comment:
On a separate note I've noticed that multiple imports from the same module are declared on separate lines, i.e.

```
from betfair import models
from betfair import constants
```

PEP 0328 recommends keeping a single import per module and using parenthesis if the line gets too long (https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses), e.g.

```
from betfair import constants, models
...
from something import (a, b, c, d, e, f
                                      g, h, i, j, k)
```

Would be in favour of changing the imports like so?